app/gui/view/component-browser/component-list-panel/src/navigator/icon.rs

```diff
@@ -0,0 +1,133 @@
+use ensogl_core::display::shape::*;
+use ensogl_core::prelude::*;
+use ide_view_component_list_panel_icons::common_part::*;
+
+use ensogl_grid_view as grid_view;
+use ide_view_component_list_panel_icons::define_icons;
+use ide_view_component_list_panel_icons::SHRINK_AMOUNT;
+
+
+
+define_icons! {
+    /// A five-pointed star.
+    pub mod star(Star) {
+        ensogl_core::define_shape_system! {
+            above = [grid_view::selectable::highlight::shape, crate::background];
```

Comment:
Section-navigator-specific icons are defined here. This line is the main reason for that, we want to have access to `crate::background`!